NEWS.rst

```diff
@@ -2,6 +2,15 @@
 Release Notes
 =============
 
+pkgdev 0.2.3 (2022-10-14)
+-------------------------
+
+- mask: fix unrelated addition of trailing whitespace (Arthur Zamarin, #98)
+
+- commit: add --distdir for manifest operations (Arthur Zamarin)
+
+- manifest: better handling of path target (Arthur Zamarin)
```

Comment:
Reference https://github.com/pkgcore/pkgdev/issues/85

Reply:
We could maybe go into specifics but given how complicated that bug is.. let's not, IMO.